react/src/redux/books/actions.js

```diff
@@ -124,6 +128,27 @@ const actionCreators = {
         dispatch({type: actionTypes.GET_BOOK_SUGGESTIONS_FAILURE});
       }
     }
+  },
+  postSuggestion: (title, author, link) => {
+    return async (dispatch) => {
+      dispatch({type: actionTypes.BOOK_LOADING});
+      const responsePost = await BookService.postBookSuggestion(title, author, link);
```

Comment:
Instead of adding more parameters, I'd receive the whole object as a parameter. What do you think ?